lib/synapse/service_watcher/zookeeper/zookeeper.rb

```diff
@@ -191,6 +216,7 @@ def watcher_callback
         # We call exists(watch), children(discover) and get(discover) with (:watch=>true)
         # This re-initializes callbacks just before get scan. So we do not miss any updates by sleeping.
         #
+        puts 'callback in invoked'
```

Comment:
I think this was a debugging comment? please remove if so, or switch to `log.debug`

Reply:
Good catch!